src/DssSpell.t.sol

```diff
@@ -904,63 +908,368 @@ contract DssSpellTest is DssSpellTestBase {
+    uint256           constant  dstGap        = 200 * MILLION * RAD;
+    uint256           constant  dstTtl        = 12 hours;
+    uint256           constant  rushThreshold = 20_000_000 * WAD;
+    uint256           constant  gushThreshold = 20_000_000 * WAD;
+    uint256           constant  cutThreshold  = 300_000 * WAD;
+
+    function testRwaConduitsPsmUpdate() public {
+        address MCD_PSM_USDC_A                  = addr.addr("MCD_PSM_USDC_A");
+        address MCD_LITE_PSM_USDC_A             = addr.addr("MCD_LITE_PSM_USDC_A");
+        address RWA015_A_OUTPUT_CONDUIT         = addr.addr("RWA015_A_OUTPUT_CONDUIT");
+
+        address[9] memory singleSwapConduits = [
+            addr.addr("RWA014_A_INPUT_CONDUIT_URN"),
+            addr.addr("RWA014_A_INPUT_CONDUIT_JAR"),
+            addr.addr("RWA014_A_OUTPUT_CONDUIT"),
+            addr.addr("RWA007_A_JAR_INPUT_CONDUIT"),
+            addr.addr("RWA007_A_INPUT_CONDUIT"),
+            addr.addr("RWA007_A_OUTPUT_CONDUIT"),
+            addr.addr("RWA015_A_INPUT_CONDUIT_JAR_USDC"),
+            addr.addr("RWA015_A_INPUT_CONDUIT_URN_USDC"),
+            addr.addr("RWA009_A_INPUT_CONDUIT_URN_USDC")
+        ];
 
-        uint256 numJobs = DssCronSequencerLike(CRON_SEQUENCER).numJobs();
+        // ----- Pre-spell sanity checks -----
+        // single swap conduits
+        _testSingleSwapRwaConduits(singleSwapConduits, MCD_PSM_USDC_A);
 
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB) , true, "TestError/old-job-not-present-in-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB_NEW), false, "TestError/new-job-already-present-in-sequencer");
+        // multi swap conduit
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_PSM_USDC_A), 1);
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_LITE_PSM_USDC_A), 0);
 
+        // ----- Execute spell -----
         _vote(address(spell));
         _scheduleWaitAndCast(address(spell));
         assertTrue(spell.done(), "TestError/spell-not-done");
 
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB), false, "TestError/old-job-not-removed-from-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).hasJob(CRON_CLIPPER_MOM_JOB_NEW), true, "TestError/new-job-not-added-to-sequencer");
-        assertEq(DssCronSequencerLike(CRON_SEQUENCER).numJobs(), numJobs, "TestError/job-amount-changed");
+        // ----- Post-spell state checks -----
+        // single swap conduits
+        _testSingleSwapRwaConduits(singleSwapConduits, MCD_LITE_PSM_USDC_A);
+
+        // multi swap conduit
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_PSM_USDC_A), 0);
+        assertEq (RwaConduitLike(RWA015_A_OUTPUT_CONDUIT).pal(MCD_LITE_PSM_USDC_A), 1);
+    }
+
+    function _testSingleSwapRwaConduits(address[9] memory conduits, address psm) internal {
```

Comment:
Maybe use `_check` instead of `_test` to keep the same convention as the base test file:
```suggestion
    function _checkSingleSwapRwaConduits(address[9] memory conduits, address psm) internal {
```

Reply:
updated in [c0b26c0](https://github.com/makerdao/spells-mainnet/pull/422/commits/c0b26c09104a5b5cb3b46d4f50856b9d5381f594)